quantus/metrics/faithfulness_metrics.py

```diff
@@ -1967,6 +1978,11 @@ def __call__(
 
         return self.last_results
 
+    @property
+    def get_auc_score(self):
+        """Calculate the area over the curve (AOC) score for several test samples."""
```

Comment:
isn't area **under** the curve calculated or am I wrong=

Reply:
Right again, I have updated it.